OracleSOASuite/dockerfiles/README.md

```diff
@@ -135,6 +136,7 @@ Sample Data will look like this:
     ADMIN_PASSWORD=Welcome1
     MANAGED_SERVER=soa_server1
     DOMAIN_TYPE=soa
+    ADMIN_HOST=slc09cwi.us.oracle.com
```

Comment:
You are adding your host address please replace for a dummy ADMIN_HOST

Reply:
Sure. I was reusing the same code which was already available for soaserver.env.list for Managed Server. I'll update at both places and push the changes.